sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -5,8 +5,9 @@
 <!-- art::Wrappers for these products are defined in CAFMaker -->
 
 <lcgdict>
-  <class name="caf::StandardRecord" ClassVersion="14">
-   <version ClassVersion="14" checksum="2636549707"/>
```

Comment:
For context: We hypothesise that the error was caused by the ClassVersion was _replaced_, rather than _incremented_.